ql/src/java/org/apache/hadoop/hive/ql/exec/Utilities.java

```diff
@@ -763,6 +763,9 @@ public static TableDesc getTableDesc(Table tbl) {
     if (tbl.getMetaTable() != null) {
       props.put("metaTable", tbl.getMetaTable());
     }
+    if (tbl.getBranchName() != null) {
+      props.put("branch_name", tbl.getBranchName());
```

Comment:
could we move String literal to constant

Reply:
Done.